ml/functions.py

```diff
@@ -0,0 +1,43 @@
+from typing import List
+
+import torch
+from torch import Tensor
+
+from ml.typing import TensorWithMask
+
+
+def concat(tensors: List[TensorWithMask], dim):
```

Comment:
The same issue with type.
```suggestion
def concat(tensors: List[TensorWithMask], dim: int):
```